generators/utilities/src/main/resources/csrc/emulator.cc

```diff
@@ -267,25 +279,25 @@ int main(int argc, char** argv)
 
 #if VM_TRACE
   Verilated::traceEverOn(true); // Verilator must compute traced signals
+#if CY_FST_TRACE
+  std::unique_ptr<VerilatedFstC> tfp(new VerilatedFstC);
+  if (vcdfile_name) {
+    tile->trace(tfp.get(), 99);  // Trace 99 levels of hierarchy
+    tfp->open(vcdfile_name);
+  }
+#else
   std::unique_ptr<VerilatedVcdFILE> vcdfd(new VerilatedVcdFILE(vcdfile));
   std::unique_ptr<VerilatedVcdC> tfp(new VerilatedVcdC(vcdfd.get()));
   if (vcdfile) {
     tile->trace(tfp.get(), 99);  // Trace 99 levels of hierarchy
     tfp->open("");
   }
-#endif
-
-  jtag = new remote_bitbang_t(rbb_port);
```

Comment:
I'm worried about this change but hopefully its addressed somewhere else.

Reply:
see my above comment about HTIF contexts. in general, i think the Simulator device should be responsible for creating the HTIF instance on the first tick(). it will unify the single-thread and multi-thread versions of the simulator.

Reply:
@abejgonzalez do we have a jtag test somewhere?

Reply:
Unfortunately we don't have a CI test for JTAG. Right now it is all manual.

Reply:
Im not sure I follow this? Where are these created now that this section is deleted?

Reply:
ill add a ticket for myself to add jtag to CI